notifier/notifier.go

```diff
@@ -383,25 +391,25 @@ func (n *Notifier) Stop() {
 
 // Describe implements prometheus.Collector.
 func (n *Notifier) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
Just one last message from me here about this: I also don't know the "right" way, and there might not be the one right way. For example, the fact that it's non-global could also be useful to me as long as I only have a small number of users, as it allows me to split up the metrics per user (via const labels). At a certain number of users, that's not feasible anymore, but it's impossible to begin with if I have globals. But yeah, happy to chat about it somewhere else.